### ISSUE TITLE
Add conda set up details to developer instructions

### ISSUE THREAD

**Issue by rt17603** [openghg/openghg#440]
For the developer set up instructions, include details of how to do with conda as well as pip.

**Comment by joe-pitt** [comment]
I attempted the following approach on 2022-11-28:

git clone -b devel https://github.com/openghg/openghg.git
conda env create --file environment.yaml 
conda activate openghg_env
conda install conda-build
conda-develop openghg

I hit two (presumably) separate issues:
1) The packages "icoscp" and "msgpack" are specified in environment.yaml but don't appear to be available through conda.
2) conda install conda-build failed with the following output:

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.

ResolvePackageNotFound: 
  - python=3.1


Issue 1) was easily solved by removing these packages from environment.yaml and installing them separately using pip install. I couldn't work out how to overcome issue 2), so I abandoned this conda approach and installed the whole thing using pip as specified in the current developer set up instructions.
